docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -38,84 +68,110 @@ Strapi currently support the following databases.
 | MariaDB    | 10.1            |
 | MongoDB    | 3.6             |
 
-## Step 2: Create a new project
+## Creating a Strapi project
 
-:::: tabs
+::: danger 🤓 CLI installation options
+The following installation guide covers the most basic installation option using the CLI. There are however other options that can be used when creating a new Strapi project:
 
-::: tab yarn
+- Using the `--quickstart` flag at the end of the command to directly create the project in quickstart mode.
+- Using the `--template` flag at the end of the command to create a project with pre-made Strapi configurations (see [Templates](templates.md)).
 
-To quickly create a blank project from scratch, run this command:
+Strapi also offers a starters CLI to create a project with a pre-made frontend application (see [our dedicated blog post](https://strapi.io/blog/announcing-the-strapi-starter-cli)).
+:::
 
-```bash
-yarn create strapi-app my-project --quickstart
-```
 
-Alternatively, to use one of our [starters](https://strapi.io/starters), run this command instead:
-
-```bash
-yarn create strapi-starter my-project <starter-url>
-```
+1. In a terminal, run the following command:
 
+:::: tabs card
+::: tab npm
+ ```sh
+  npx create-strapi-app my-project
+  ```
+::: 
+::: tab yarn
+ ```sh
+  yarn create strapi-app my-project
+ ```
 :::
 
-::: tab npx
+2. Choose an installation type:
+   * `Quickstart (recommended)`, which uses the default database (SQLite)
+   * `Custom (manual settings)`, which allows to choose your preferred database
 
-To quickly create a blank project from scratch, run this command:
+3. (Custom installation type only) Among the list of databases, choose a database for your Strapi project.
 
-```bash
-npx create-strapi-app my-project --quickstart
-```
+4. (Custom installation type only) Name your project's database.
+
+## Running Strapi
 
-Alternatively, to use one of our [starters](https://strapi.io/starters), run this command instead:
+To start the Strapi application, run the following command in the project folder:
 
+:::: tabs card
+::: tab npm
 ```bash
-npx create-strapi-starter my-project <starter-url>
+npm run develop
 ```
-
-:::
-
-::::
-
-::: tip
-If you want to use specific database, you don't have to use the `--quickstart` flag. The CLI will let you choose the database of your choice. The `--quickstart` flag sets the database to SQLite.
 :::
 
-:::tip
-By default, `create-strapi-app` will generate an empty Strapi project. If you want an application that is pre-configured for a specific use case, see the [Templates section](/developer-docs/latest/setup-deployment-guides/installation/templates.md).
+::: tab yarn
+```sh
+yarn develop
+```
 :::
 
-:::tip
-When using a starter, the project can be created from a specific branch, for example with a url like this: `https://github.com/strapi/strapi-starter-gatsby-blog/tree/<my-branch>`.
-:::
+<!-- ### Creating a project with starters
```

Comment:
What is the plan with Starters? It's commented.

Reply:
On va lui dire que pour l'instant on fait pas de guide dédié sur les starters puisque c'est une documentation nouvelle qu'on a pas, et pour l'instant le but du projet était surtout de réécrire ce qui existait déjà.

For now, the plan is to not create a dedicated starters guide. This is going to be a new documentation.
The goal of this PR was mostly to rewrite the existing content.

The commented parts are going to be removed, thank you!